### DISCUSSION TITLE
One out of three lights keeps losing sync (the same one every time)

### DISCUSSION THREAD

**Discussion by jh20001** [basnijholt/adaptive-lighting#333] [Q&A]
So I have three Hue lights that are added to this that fluctuate throughout the day with the sun. It's an awesome effect. I set the minimum brightness to 100 as I felt the dimming made it too hard to see as I still work in there later in the day and need light. Everything is great outside of one bug. One of the three lights keeps losing sync and will be a different color. 

It still has connectivity to HA as I can turn it off and back on again separately from HA. It will then after so many seconds join back into sync with the rest of the lights. Just to make sure I wasn't losing my mind and to rule out all possibilities, I shuffled the bulbs around the room. No matter where that one Hue bulb is, it is the only one that loses sync.

# Sunrise Sunset Office Light Settings
adaptive_lighting:
  lights:
    - light.office_hue_color_1
    - light.office_hue_color_2
    - light.office_hue_color_3
  min_brightness: 100

**Comment by basnijholt** [comment]
Could you share the settings of your adaptive lightning instance?

I think adaptive lighting might think you are taking over control for some of the lights.

**Comment by jh20001** [comment]
The only other settings are in the image below. Could it be the take_over_control option that is enabled? I figured I'd want it to take control. The only other thing those three lights go through is an on/off switch on the wall throughout the day as they don't always remain on. All three are controlled together by that wall switch as it controls an outlet they are all connected to. That individual Hue bulb isn't being controlled by anything else. There is also nothing enabled in the Hue app/bridge to adjust the lights for any reason.

![image](https://user-images.githubusercontent.com/6210275/188285887-4fbfe6c8-7110-4dec-a237-41d0764562a5.png)

**Comment by basnijholt** [comment]
Could you try turning off `take_over_control` and see if the problem persists?

I am not saying this is the solution, but it will bring us to the right place to identify where the problem is coming from.

**Reply by jh20001** [reply]
I'll give it a try and circle back. I don't know how much time I will be spending there the next few days, but I'll squeeze what I can :)

**Reply by jh20001** [reply]
Sadly, that didn't work. Upon turning the light on, I walked out of the room and came back in and two were updated to where they should be and the one that always misbehaves did not. 
![20220904_190409](https://user-images.githubusercontent.com/6210275/188346695-752e5a93-fee5-4104-8a51-8b099cda5390.jpg)

**Reply by basnijholt** [reply]
Ok good to know. Then we know it's not because Adaptive Lightning thinks the light is manually controlled

**Reply by jh20001** [reply]
I guess I'll just cook up an automation that sets them to one temp during the day and another at night. *shrugs*
It was worth a try though as AL seemed like a good idea (edit: is a great idea). Maybe it's just my luck. I always seem to find the one special needs device in the bunch. Maybe this means I should never have kids or one might pop out with something just as odd ("are.....those feathers?...")

**Comment by th3w1zard1** [comment]
@jh20001 if you're still experiencing this, please set your logging to debug (see the readme) and post some log entries here when you notice it happening, it'd make it a lot easier to debug this issue.
